app/components/review_draft/draft/view.html.erb

```diff
@@ -32,6 +32,14 @@
           component.row(**row_helper(@trainee, :school_details))
         end
 
+        if FeatureService.enabled?("funding")
+          if cannot_start_funding
+            component.row(**row_helper(@trainee, :funding_inactive))
```

Comment:
Would this be neater if we had a helper method that would just return the options for the row helper ie. `:funding_inactive` or `funding_active`? then this line could be 
```
component.row(**row_helper(@trainee, funding_options(@trainee)))
```
since the only thing we need to calculate `cannot_start_funding?` and the feature flag are the trainee. Then we won't need to bother passing `cannot_start_funding` into the review draft components. What do you think?

Reply:
Much tidier! I added it to a new `FundingHelper`.